integration_tests/sdk/setup_integration.py

```diff
@@ -325,6 +325,24 @@ def _fetch_integration_credentials(section: str, name: str) -> Dict[str, Any]:
     return test_credentials[section][name]
 
 
+def lazy_configuration(name: str) -> bool:
```

Comment:
The logical jump from `enable_previews` to "should have lazy execution" actually took me a while to get. I think it would be good to explain in the fixture why external compute engines need to have their executions set to lazy for previews.

I think part of the confusion was that the docstrings and name of this function attempt to explain things beyond the scope of it. I've prefer if that explanation was in the caller, and this method was focused on just checking if `enable_preview` was set - probably renaming this to `is_preview_enabled()` or something.